sources/_sector/dex/trades/lens/_sources.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+
+sources:
+  - name: uniswap_v3_lens
+    tables:
+      - name: uniswapv3pair_evt_swap
```

Comment:
looks like the swap events haven't been decoded yet, can't find in data explorer
factory is here
`uniswap_v3_lens.uniswapv3factory_evt_poolcreated`

Reply:
model should fail in CI